demisto_sdk/commands/unify/unifier.py

```diff
@@ -338,3 +351,46 @@ def clean_pwsh_code(script_code):
         script_code = script_code.replace(". $PSScriptRoot\\demistomock.ps1", "")
         script_code = script_code.replace(". $PSScriptRoot\\CommonServerPowerShell.ps1", "")
         return script_code
+
+    def get_pack_path(self):
+        return self.package_path.split('Integrations')[0]
+
+    def is_contributor_pack(self):
```

Comment:
The name does not really describe this method.
I would suggest something like `get_contributor_data` or something of that sort.
I would expect this name to return an only boolean result.
This change will also affect the docstring description.

Reply:
i agree, should have one for the boolean, and the other one for the data.

the data one should be called from the add contributors data